lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -197,6 +197,22 @@ def instructor_dashboard_2(request, course_id):
     if certs_enabled and access['admin']:
         sections.append(_section_certificates(course))
 
+    def get_course_blocks(course_key, category):
+        """
+        Retrieve all XBlocks in the course for a particular category.
+
+        Returns only XBlocks that are published and haven't been deleted.
+        """
+        # Note: we need to check if found components have been orphaned
```

Comment:
Is this comment still applying? @andrey-canon

Reply:
I have deleted some xblocks and it returns the right number, I think this method is not necessary